src/bin/interactive.rs

```diff
@@ -99,7 +99,7 @@ pub(crate) async fn run_loop(
                             Ok(res) => {
                                 println!("\r{}", res);
                                 if let RunResult::Tx(hash) = res {
-                                    let txh = format!("{hash:x}");
+                                    let txh = hex::encode(hash.to_bytes());
```

Comment:
I find `txh` as variable name a bit ambiguous. Could be `(block)height`, could be `hash`, could be `header`.

Reply:
changed to `tx_id` => TransactionID